data/strings/strings.txt

```diff
@@ -1806,7 +1806,7 @@
     zh-Hant = 收藏夾
 
   [bookmarks]
-    comment = "Bookmarks" dialog title, please also edit it in iphone/plist.txt
+    comment = "Bookmarks" dialog title (kept for compatibility, please also edit it in iphone/plist.txt)
```

Comment:
Did you identify all places where is it used on Android? If the only usage is on iOS, then comment should explicitly mention that.

Reply:
? What is "compatibility"? This string is either used in the code, or is not used. Let's find it and specify explicitly.